src/test/java/org/cactoos/http/HtTimedWireTest.java

```diff
@@ -60,20 +58,19 @@ public void worksFine() throws Exception {
         );
     }
 
-    @Ignore("see BlockingSocketServer todo for #87")
     // @checkstyle MagicNumberCheck (1 line)
     @Test(expected = TimeoutException.class, timeout = 1000)
+    @SuppressWarnings("PMD.AvoidUsingHardCodedIP")
     public void failsAfterTimeout() throws Exception {
-        // @checkstyle MagicNumberCheck (1 line)
+        // @checkstyle MagicNumberCheck (2 lines)
         final long timeout = 100;
-        try (BlockingSocketServer server = new BlockingSocketServer()) {
-            new HtTimedWire(
-                new HtWire(
-                    server.address().getHostAddress(),
-                    server.port()
-                ),
-                timeout
-            ).send(new InputOf("unused"));
-        }
+        final int port = 80;
+        new HtTimedWire(
+            new HtWire(
+                "192.168.0.0",
```

Comment:
@iakunin are we sure this one can never be routable? Why not use the example one in the SO answer (`10.255.255.1`)?

Reply:
@victornoel you're right: using `10.255.255.1` is more obvious, cause this one is in SO answer.

Replaced.